### PR TITLE
docs - describe the new resource group CPUSET feature

### PR DESCRIPTION
**Pull request by lisakowen**
document the new resource group CPUSET feature (reserving specific CPU cores for a resource group) - PR #5023:
- CREATE/ALTER RESOURCE GROUP reference page updates
- sgml docs reference page updates
- add new capability 7 (cpuset):
    - pg_resgroupcapability
    - gp_resgroup_config
    - gp_toolkit
- "using resource groups page" updates:
    - expand on the new CPU limit
	- add cpuset {} block to RG cgroups config section
	- misc other areas - qualify CPU_RATE_LIMIT/CPUSET

question - any new CPU-related FAQs or adjustments to the current CPU info for the new CPUSET feature (http://docs-gpdb-review-staging.cfapps.io/review/admin_guide/workload_mgmt_resgroups.html#topic777999)?

(i will submit a separate PR for 5X_STABLE.  some of the "using" and gp_resgroup_config/gp_toolkit content will differ in 5x.)

links to doc review site:
- http://docs-gpdb-review-staging.cfapps.io/review/ref_guide/sql_commands/CREATE_RESOURCE_GROUP.html
- http://docs-gpdb-review-staging.cfapps.io/review/ref_guide/sql_commands/ALTER_RESOURCE_GROUP.html
- http://docs-gpdb-review-staging.cfapps.io/review/ref_guide/system_catalogs/pg_resgroupcapability.html
- http://docs-gpdb-review-staging.cfapps.io/review/ref_guide/system_catalogs/gp_resgroup_config.html
- http://docs-gpdb-review-staging.cfapps.io/review/ref_guide/gp_toolkit.html#topic27x
- http://docs-gpdb-review-staging.cfapps.io/review/admin_guide/workload_mgmt_resgroups.html#topic833971717

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml [hunk 1]
```diff
@@ -219,23 +240,28 @@
          higher than .9. Doing so may result in high workload
          queries taking near all CPU resources, potentially starving Greenplum
          Database auxiliary processes.</note>
+      <p>(While <codeph>gp_resource_group_cpu_limit</codeph> applies to the CPU resources of all CPU cores on the host, this configuration parameter setting has no effect on resource groups that you configure with <codeph>CPUSET</codeph>.)</p>
```
Comment (gaos1) [inline gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: This is not accurate. gp_resource_group_cpu_limit take affect to both cpu_rate_limit and cpuset.  or gpdb all resource group cpu usage including cpu_rate_limit and cpuset ones are under the overall limit of gp_resource_cpu_limit of the entire system.

**Comment by lisakowen** [comment]
i reorganized the content a bit, @gaos1 : moved the gp_resource_group_cpu_limit discussion from the "assigning cpu resources by percentages" section to the "cpu limits" overview section.

File: gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml [hunk 2]
```diff
@@ -219,23 +228,48 @@
...
+            <li><codeph>admin_group</codeph> and <codeph>default_group</codeph> require at least one CPU core. When all CPU cores are reserved, Greenplum Database assigns CPU core 0 to these default groups. In this situation, the resource group to which you assigned CPU core 0 shares the core with <codeph>admin_group</codeph> and <codeph>default_group</codeph>.</li>
+            <li>If you restart your Greenplum Database cluster with one node replacement and the node does not have enough cores to service all <codeph>CPUSET</codeph> resource groups, the groups are automatically assigned CPU core 0 to avoid system start failure.</li></ul></li>
+          <li>Use the lowest possible core numbers when you assign cores to resource groups. If you replace a Greenplum Database node and the new node has fewer CPU cores than the original, or if you back up the database and want to restore it on a cluster with nodes with fewer CPU cores, the operation may fail. For example, if your Greenplum Database cluster has 16 cores, assigning cores 1-7 is optimal. If you create a resource group and assign CPU core 9 to this group, database backup to an 8 core node will fail.</li></ul></p>
```
Comment (gaos1) [inline gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: "database backup to an 8 core node will fail" should be "database restore to an 8 core node will fail"

**Comment by gaos1** [comment]
LGTM. Thanks.

**Pull request merged**